projects/addon-editor/constants/default-editor-tools.ts

```diff
@@ -10,6 +10,8 @@ export const defaultEditorTools = [
     TuiEditorTool.List,
     TuiEditorTool.Quote,
     TuiEditorTool.Link,
+    TuiEditorTool.Anchor,
+    TuiEditorTool.Attach,
```

Comment:
Do we really need to add it here ?

Is it not breaking changes ?

Reply:
fixed